chapters/background.tex

```diff
@@ -1,4 +1,13 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+Since Viola and Jones proposed "Rapid Object Detection using a Boosted Cascade of Simple Features" in 2001, their concept is the basis of most face detection approaches.
```

Comment:
Im pritty sure latex does not like quotes like this. Also, in line 7 and 9, you use different kind of quotation marks.
For english quotes in latex, please use ` `` ` for the first and `''` (two single quotes) as quotation marks. See here:
https://tex.stackexchange.com/questions/531/what-is-the-best-way-to-use-quotation-mark-glyphs